### ISSUE TITLE
TypeError: 'java.lang.String[]' object is not callable

### ISSUE THREAD

**Issue by Zaoyee** [jpype-project/jpype#960]
Hi, thanks for the package. It helps a lot for my project.
I met a problem in my project and unbale to find reason anywhere. 

I was trying to call main function of the `abc.jar`, so I have to pass a `String[]`.
As mentioned in JPype Docs, string array can be defined as following
```python

from com.abc.bce import xxx

args = JString[1](JString("hello_world"))
xxx.main(args)
```

And I got error like this , I was wondering why
```
TypeError: 'java.lang.String[]' object is not callable
```
btw, Is there any other way to assign the value to the JString array, 
somthing like this would be nice
```python
args = JString[1]
args[0] = JString("wtf-lol-:>") ## I actually got the error by doing this way
```

Thanks for the help,
Respect

**Comment by pelson** [comment]
I don't know how to do it with the new syntax (that you are showing), but it can be done with the old syntax, as documented at https://jpype.readthedocs.io/en/latest/quickguide.html?highlight=jarray#arrays, with:

```
>>> s_arr = jp.JArray(jp.JString, 1)(['one', 'two', 'three'])
>>> type(s_arr)
<java class 'java.lang.String[]'>
>>> print(list(s_arr))
['one', 'two', 'three']
```

**Comment by Thrameos** [comment]
This one was fixed by the array bug.  It was making arrays of type String[][] rather than String[].   It will be fixed in 1.3.
